src/core_plugins/kibana/public/management/react/store/index.js

```diff
@@ -0,0 +1,9 @@
+import { createStore, compose, applyMiddleware } from 'redux';
+import { rootReducer } from './reducers';
+import thunk from 'redux-thunk';
+
+const initialState = {};
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const store = composeEnhancers(applyMiddleware(thunk))(createStore)(rootReducer, initialState);
+
+export { store };
```

Comment:
`export const store` above (not important here, just mentioning as it's done in lots of places)